ouroboros-network/src/Ouroboros/Cardano/Network/PeerSelection/PeerChurnArgs.hs

```diff
@@ -0,0 +1,18 @@
+module Ouroboros.Cardano.Network.PeerSelection.PeerChurnArgs where
+
+import Cardano.Network.ConsensusMode (ConsensusMode)
+import Cardano.Network.PeerSelection.Bootstrap (UseBootstrapPeers)
+import Control.Concurrent.Class.MonadSTM.Strict
+import Ouroboros.Cardano.Network.Types (ChurnMode)
+import Ouroboros.Network.BlockFetch.ConsensusInterface (FetchMode)
+import Ouroboros.Network.PeerSelection.Governor.Types (PeerSelectionTargets)
+
+data CardanoPeerChurnArgs m =
+  CardanoPeerChurnArgs {
+    cpcaModeVar            :: StrictTVar m ChurnMode
+  , cpcaReadFetchMode      :: STM m FetchMode
+  , cpcaGenesisPeerTargets :: PeerSelectionTargets
+  , cpcaReadUseBootstrap   :: STM m UseBootstrapPeers
+  , cpcaConsensusMode      :: ConsensusMode
+  }
```

Comment:
or do we want to use a qualified import as I did in some PRs and call this type: `Arguments` and drop all the `cpca` prefixes?

I think qualified imports are much nicer, easier to find where something comes from (if `hls` is down), things like `Churn.Arguments` or `Cardano.Churn.Arguments` are pretty clear what it is (unless one is using `CC.Arguments` :grin:)

Reply:
We can do that sure :)